pkg/providers/cloudstack/cloudstack.go

```diff
@@ -804,6 +803,11 @@ func buildTemplateMapCP(clusterSpec *cluster.Spec, controlPlaneMachineSpec, etcd
 		values["awsIamAuth"] = true
 	}
 
+	auditPolicy, err := common.GetAuditPolicyYaml(clusterSpec.Cluster.Spec.KubernetesVersion)
```

Comment:
There will be some unit test failures here, since we assert on yaml files which are likely to change if we move to a serialized go struct instead of the previous template. Also, I suspect that change itself will trigger a rolling upgrade on the workload clusters during a management cluster upgrade still, which was what I thought we were trying to avoid with this PR.

Reply:
Yes, we are planning on now keeping the same yaml for versions before 1.23

Reply:
Check it out here https://github.com/aws/eks-anywhere/pull/3672